express-GET/server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const fs = require('fs');
+
+const port = 3000;
+const app = express();
+
+
+app.get('/llip', (req, res) => {
+  const getName = (err, file) => {
+    const result = [];
+    file.forEach(name => {
+      if (name !== 'emmanuel') {
+        result.push(name);
+      }
+    });
+    res.send(result+'\r\n');
```

Comment:
whats the point of sending back `song emmanuel joe \r\n`? Like why put `\r\n` at the very end of the string?